data/json/furniture_and_terrain/furniture-appliances.json

```diff
@@ -501,7 +501,7 @@
     "id": "f_drophammer",
     "name": "blacksmith drop hammer",
     "symbol": "#",
-    "description": "An anvil with a large metal hammer suspended above it in a metal framework.  If it were working, it would be useful for shaping softened metal plates, though now it is only useful for parts.",
+    "description": "An anvil with a large metal hammer suspended above it in a metal framework.  I would be useful for shaping softened metal plates, or you could use the anvil by itself.",
```

Comment:
```suggestion
    "description": "An anvil with a large metal hammer suspended above it in a metal framework.  It would be useful for shaping softened metal plates, or you could use the anvil by itself.",
```